lua/orgmode/config/init.lua

```diff
@@ -413,33 +414,25 @@ function Config:respect_blank_before_new_entry(content, option, prepend_content)
   return content
 end
 
----@param amount number
----@return string
-function Config:get_indent(amount)
-  if self.org_adapt_indentation then
-    return string.rep(' ', amount)
+---Check if buffer should apply indentation
+---@param bufnr? number
+function Config:should_indent(bufnr)
+  if bufnr > -1 and vim.b[bufnr].org_indent_mode then
```

Comment:
Since `bufnr` is optional according to the annotation on line 418, ideally we'd also check if `bufnr` exists:

```suggestion
  if bufnr and bufnr > -1 and vim.b[bufnr].org_indent_mode then
```

This will keep callers who don't provide a `bufnr` from having their head explode when `nil` is compared `-1`. 

Alternatively, you could make providing a `bufnr` non-optional. It seems all other callers of this function currently *do* provide a `bufnr` -- seems it may be better to require it.

Reply:
The idea was to have it mandatory. Updated the annotation.